qmk_gui.py

```diff
@@ -378,20 +371,24 @@ def find_devs(show, verbose):
                 print("Usage Page not matching")
             continue
 
-        fw_ver = device_dict["release_number"]
-
         if (os.name == 'nt' and device_dict['usage_page'] == RAW_USAGE_PAGE) or verbose:
             if show:
+                manufacturer = device_dict["manufacturer_string"]
                 print(f"Manufacturer: {manufacturer}")
+                product = device_dict["product_string"]
                 print(f"Product:      {product}")
-                print("FW Version:   {}".format(format_fw_ver(fw_ver)))
+                fw_ver = device_dict["release_number"]
+
+                print(f"FW Version:   {format_fw_ver(fw_ver)}")
+                sn = device_dict['serial_number']
                 print(f"Serial No:    {sn}")
 
-            if verbose:
-                print(f"VID/PID:      {vid:02X}:{pid:02X}")
-                print(f"Interface:    {interface}")
-                # TODO: print Usage Page
-                print("")
+        if verbose:
+            pid = device_dict["product_id"]
+            print(f"VID/PID:      {vid:02X}:{pid:02X}")
+            print(f"Interface:    {interface}")
+            # TODO: print Usage Page
+            print("")
```

Comment:
I actually don't always want to print this, unless the above condition is true. Keep it as it was please.

```suggestion
            if verbose:
                pid = device_dict["product_id"]
                print(f"VID/PID:      {vid:02X}:{pid:02X}")
                print(f"Interface:    {interface}")
                # TODO: print Usage Page
                print("")
```